app/components/dropdowns/dropdown-single-select/component.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { useSelect } from 'downshift';
+import ARROW_DOWN_SVG from 'svgs/ui/arrow-down.svg';
+import Icon from 'components/icon';
+import cx from 'classnames';
+
+const THEME = {
+  dark: {
+    container: 'text-white bg-gray-800 border-2 rounded-3xl py-1.5',
+    open: 'border-2 border-primary-400 text-base',
+    closed: 'border-gray-400 text-gray-400',
+    icon: {
+      closed: 'text-white',
+      open: 'fill-primary transform rotate-180',
+      disabled: 'text-gray-400',
+    },
+    item: {
+      base: 'text-gray-300',
+      highlighted: 'bg-gray-700 text-white',
+    },
+  },
+  light: {
+    container: 'text-gray-600 bg-white border-2 rounded-3xl py-1.5',
+    open: 'border-2 border-primary-400 text-base',
+    closed: 'border-gray-400 text-gray-400',
+    icon: {
+      closed: 'text-gray-600',
+      open: 'fill-primary transform rotate-180',
+      disabled: 'text-gray-400',
+    },
+    item: {
+      base: 'text-gray-400',
+      highlighted: 'bg-gray-100 text-gray-800',
+    },
+  },
+  states: {
+    none: '',
+    error: 'border-red-500',
+    valid: 'border-green-500',
+  },
+};
+
+interface Option {
+  label: string;
+  value: string | number;
+  checkbox?: boolean;
+  disabled?: boolean;
+}
+
+export interface SingleSelectProps {
+  theme?: 'dark' | 'light';
+  disabled?: boolean;
+  prefix?: string;
+  placeholder?: string;
+  options?: Array<Option>;
```

Comment:
```suggestion
  options?: Option[];
```

Is this the same? If it is, please change it because consistency.